src/settings.ts

```diff
@@ -441,6 +451,19 @@ export const ROOM_TEMPLATES: RoomTemplate[] = [
       },
     ],
   },
+  {
+    template: VenueTemplate.embeddable,
+    name: "Embeddable",
+    description: "Add an embeddable with an embedded iframe in the center",
+    icon: "/venues/pickspace-thumbnail_auditorium.png",
```

Comment:
[NB] Personally I would leave this `undefined` (is that even supported for this?) rather than re-using the same icons/etc.

Related comments/questions:

- https://github.com/sparkletown/internal-sparkle-issues/issues/505#issuecomment-799387064
- https://github.com/sparkletown/internal-sparkle-issues/issues/505#issuecomment-799713071

Reply:
changed to an empty string in https://github.com/sparkletown/sparkle/commit/c95c932f2c2b10cfa98f09884f67ddfd551ced42